### ISSUE TITLE
Hyperparameters for more datasets

### ISSUE THREAD

**Issue by zzzzzx-1115** [ningshuliang/MIMO-VP#4]
Thanks for your awesome MIMO-VP!

I am trying to train MIMO on datasets besides Moving MNIST, so could you please help me specify the hyperparameters for experiments on the Human3.6M, weather and KITTI datasets?

Thanks for your help!

**Comment by zzzzzx-1115** [comment]
Besides, it seems that the released codes are specialized for Moving MNIST dataset? I notice there are some parameters are hard coded, e.g., the _index_ in function _forward_ in class _Transformer_ uses _20_, which i guess is the total length of Moving MNIST videos?

**Comment by ningshuliang** [comment]
Yes, 20 is the total length of video clips. if you want to use it in Human3.6M and KITTI datasets, just set it to 8 and 10 respectively.
